<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recon_configuration_page_exif_metadata</class>
 <widget class="QWidget" name="recon_configuration_page_exif_metadata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_select_all">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_search">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_exif_metadata">
       <property name="styleSheet">
        <string notr="true">QTableWidget {


/*selection-background-color: #87CEFF;*/
 /*selection-background-color: #0069EB; */
selection-color:white;
 }


QTableWidget::item{ 
	 height: 26px; 
 }




QTableWidget::item:selected:!active {
    color:white;
    background-color:  #0069EB;
}


 
                                QTableWidget {
                                 selection-background-color: #0069D7;
                                    } 




QScrollBar{
		width:9px;
}


QTableWidget::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
QTableWidget::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}





QTableWidget::indicator {
width: 13px;
height: 13px;
}</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>D</string>
        </property>
        <property name="whatsThis">
         <string extracomment="Categories Display"/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>M</string>
        </property>
        <property name="whatsThis">
         <string extracomment="Show In Metadata"/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>R</string>
        </property>
        <property name="whatsThis">
         <string extracomment="Show In Report"/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Attribute</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
